lib/src/gnav.dart

```diff
@@ -70,6 +70,12 @@ class _GNavState extends State<GNav> {
     selectedIndex = widget.selectedIndex;
   }
 
+  @override
+  void didUpdateWidget(GNav oldWidget) {
+    super.didUpdateWidget(oldWidget);
+    if(widget.selectedIndex != oldWidget.selectedIndex) selectedIndex = widget.selectedIndex;
```

Comment:
Wouldn't it be better if there were curly braces? I mean, the logic still the same but then there isn't a lot into a single line.

Reply:
@Ascenio has been updated, as per your request.